app/uk/gov/hmrc/eusubsidycompliancefrontend/syntax/EitherTSyntax.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eusubsidycompliancefrontend.syntax
+
+import cats.data.EitherT
+
+import scala.concurrent.{ExecutionContext, Future}
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.Error
+
+object EitherTSyntax {
```

Comment:
Included this file just for a review, if we want to use EitherT in future. I am not currently using it.  I can remove it if you want :)

Reply:
This is good to see. That's exactly why I added the `OptionTSyntax` so we could look into moving to something like this.